### PR TITLE
WalletConnect

### LINKED ISSUE #679: WalletConnect support

### PR DESCRIPTION
**Pull request by michael1011**
Closes #679 

Available only on the mainnet Vercel deployment

@kilrau needs translations

### CODE REVIEW

File: src/components/WalletConnect.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                return;
+            }
+
+            const address = createdKit().getAddress();
+            const provider = new BrowserProvider(
+                await createdKit().getUniversalProvider(),
+            );
+
+            WalletConnectProvider.resolveClosePromise(provider, address);
+        });
```
Comment (jackstar12) [inline src/components/WalletConnect.tsx:85]: Why not put that into the above resource?

File: src/components/WalletConnect.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            metadata: {
+                name: "Boltz",
+                description: "Boltz Web App",
+                url: getLocation(),
+                icons: [`${getLocation()}/android-chrome-512x512.png`],
+            },
+            features: {
+                email: false,
+                socials: false,
+                analytics: true,
```
Comment (jackstar12) [inline src/components/WalletConnect.tsx:61]: What kinds of analytics is this about?

**Review by kilrau — APPROVED** [review]
tACK (Beexo)

File: src/components/WalletConnect.tsx (hunk 2 above)
Comment (michael1011) [reply src/components/WalletConnect.tsx:61]: https://docs.reown.com/appkit/react-native/cloud/analytics

**Pull request merged**